modules/apps/batch-engine/batch-engine-test/src/testIntegration/resources/com/liferay/batch/engine/internal/test/dependencies/batch1/batch1/batch-engine.json

```diff
@@ -2,7 +2,9 @@
 	"callbackURL": null,
 	"className": "com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition",
 	"companyId": 90003,
-	"parameters": null,
+	"parameters": {
+		"fileName": "/batch1/export.json"
```

Comment:
Please don't do it this way! You will break every single JSON already in a client extension.

Reply:
Also, why do you need to put this in the JSON, the file is in the OSGi module, just use it's path!

Reply:
That is just for the sake of testing. That value is not real, and could have been any string so that we can use it to verify the order in the test.

Reply:
Yeah, the value is inserted in each test file and then, it is also asserted just in the test, as it is a Map of parameters. I agree that the chosen name could be a bit confusing. I am going to think about a more clear way. Anyway, it is not breaking anything or any client